### PR TITLE
Ngwmn 1910 connect stats method wiki page to ngwmn ui

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
JIRA task: https://internal.cida.usgs.gov/jira/browse/NGWMN-1910

Description
-----------
This task adds a stats methods page to the UI that pulls stats documentation from a confluence wiki page.  It also makes some adjustments to the waterlevel stats display depending on whether or not monthly stats should be displayed.

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: server/ngwmn/views.py [hunk 1]
```diff
@@ -43,6 +43,12 @@ def version():
         'version': __version__
     })
 
+@app.route('/statistics-methods', methods=['GET'])
```
Comment (mbucknell) [inline server/ngwmn/views.py]: The route should end with '/'

File: server/ngwmn/templates/site_location.html [hunk 2]
```diff
@@ -212,7 +212,11 @@ <h1>{{ feature.SITE_NAME }}</h1>
                         <tbody>
                             <tr>
                                 <td>{{ stats.overall.min_value }}</td>
+                                {% if stats.overall.median %}
+                                <td>{{ stats.overall.median }}</td>
+                                {% elif stats.overall.median_value %}
                                 <td>{{ stats.overall.median_value }}</td>
+                                {% endif %}
```
Comment (duselmann) [inline server/ngwmn/templates/site_location.html]: We need to look into this as it turns out now seeing this in context. This is working when there is a value.

**Comment by duselmann** [comment]
I do not know if we are supposed to update this file assets/package-lock.json

File: server/ngwmn/templates/statistics_methods.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="usa-width-one-whole">
+        <h1>Testing new NGWMN Statistics Methods Page!</h1>
```
Comment (duselmann) [inline server/ngwmn/templates/statistics_methods.html]: I do not think we can merge with this placeholder text.

**Comment by mbucknell** [comment]
Regarding package-lock.json. Updating the version is fine. Not sure the other lines should have changed from https to http.

**Comment by duselmann** [comment]
Thanks Mary, I was unsure. I recalled Dan cringing when I committed that file. It makes sense to commit it to me.

**Comment by mbucknell** [comment]
There are occasions where it's ok to update package-lock.json. Specifically, if dependabot finds a security risk in one of your child dependencies. Generally, I just merge the dependabot recommendation.

File: server/ngwmn/services/ngwmn.py [hunk 4]
```diff
@@ -570,6 +582,8 @@ def get_statistics(agency_cd, site_no):
                         month_stats = monthly[str(month_num)]
                         month_stats['month'] = month_abbr
                         stats['monthly'].append(month_stats)
+                for month in stats['monthly']:
+                    replace_null_values(month, substitution)
```
Comment (duselmann) [inline server/ngwmn/services/ngwmn.py]: there might be a mix of spaces and tabs?

Comment (mbucknell) [reply server/ngwmn/services/ngwmn.py]: I think it is okay. I had to look at the entire file though.

Comment (duselmann) [reply server/ngwmn/services/ngwmn.py]: thanks for checking, Mary.

**Pull request merged**